Makefile

```diff
@@ -1,16 +1,16 @@
 .PHONY: all binary ut clean
 
 BUILD_DIR=build_calico_rkt
-BUILD_FILES=Dockerfile requirements.txt
+BUILD_FILES=$(BUILD_DIR)/Dockerfile $(BUILD_DIR)/requirements.txt
```

Comment:
since the structure was copied from other repos, calico_rkt inherited two Dockerfile/req files. I deleted one of them and decided to keep them out of root. I suppose I got rid of the wrong one.